packages/nextra-theme-docs/src/components/search.tsx

```diff
@@ -241,85 +117,77 @@ export function Search({
     [handleChange]
   )
 
-  return (
-    <div className={cn('nextra-search _relative md:_w-64', className)}>
-      {renderList && (
-        <div className="_fixed _inset-0 _z-10" onClick={() => setShow(false)} />
-      )}
+  const handleSelect = useCallback(
+    (route: string) => {
+      if (route) {
+        router.push(route)
+        setMenu(false)
+      }
+    },
+    [router, setMenu]
+  )
```

Comment:
Should the combobox value be cleared after navigating to a new page? Currently, the implementation retains the value.

Reply:
Added!